reference-architecture/gce-cli/gcloud.sh

```diff
@@ -344,6 +344,9 @@ gcloud --project "$GCLOUD_PROJECT" compute ssh "cloud-user@${OCP_PREFIX}-bastion
         --enable=\"rhel-7-server-extras-rpms\" \
         --enable=\"rhel-7-server-ose-${OCP_VERSION}-rpms\";
 
+    if ! rpm -q epel-release; then
+      yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    fi
```

Comment:
You need to enable epel and install libcloud after atomic-openshift-utils if not then ansible-2.3 is installed due to epel.

```
[root@trunks aws-ansible]#     yum install -y git python-libcloud atomic-openshift-utils
Loaded plugins: product-id, search-disabled-repos, subscription-manager
Package git-1.8.3.1-6.el7_2.1.x86_64 already installed and latest version
Package python-libcloud-0.20.1-1.el7.noarch already installed and latest version
Resolving Dependencies
--> Running transaction check
---> Package atomic-openshift-utils.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
--> Processing Dependency: openshift-ansible-playbooks = 3.5.60 for package: atomic-openshift-utils-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Running transaction check
---> Package openshift-ansible-playbooks.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
--> Processing Dependency: openshift-ansible = 3.5.60 for package: openshift-ansible-playbooks-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Processing Dependency: openshift-ansible-callback-plugins = 3.5.60 for package: openshift-ansible-playbooks-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Processing Dependency: openshift-ansible-filter-plugins = 3.5.60 for package: openshift-ansible-playbooks-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Processing Dependency: openshift-ansible-lookup-plugins = 3.5.60 for package: openshift-ansible-playbooks-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Processing Dependency: openshift-ansible-roles = 3.5.60 for package: openshift-ansible-playbooks-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Running transaction check
---> Package openshift-ansible.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
--> Processing Dependency: openshift-ansible-docs = 3.5.60 for package: openshift-ansible-3.5.60-1.git.0.b6f77a6.el7.noarch
--> Processing Dependency: ansible >= 2.2.0.0-1 for package: openshift-ansible-3.5.60-1.git.0.b6f77a6.el7.noarch
---> Package openshift-ansible-callback-plugins.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
---> Package openshift-ansible-filter-plugins.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
---> Package openshift-ansible-lookup-plugins.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
---> Package openshift-ansible-roles.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
--> Running transaction check
---> Package ansible.noarch 0:2.3.0.0-3.el7 will be installed
---> Package openshift-ansible-docs.noarch 0:3.5.60-1.git.0.b6f77a6.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===================================================================================================================================================================================================================
 Package                                                       Arch                              Version                                               Repository                                             Size
===================================================================================================================================================================================================================
Installing:
 atomic-openshift-utils                                        noarch                            3.5.60-1.git.0.b6f77a6.el7                            rhel-7-server-ose-3.5-rpms                            234 k
Installing for dependencies:
 ansible                                                       noarch                            2.3.0.0-3.el7                                         epel                                                  5.7 M
```